### PR TITLE
Change short trace message display format

### PR DESCRIPTION
**Pull request by poornas**
abridge function name,time format and
add client ip in the output

### CODE REVIEW

File: cmd/admin-trace.go [hunk 1]
```diff
@@ -154,30 +152,36 @@ type responseInfo struct {
...
+	op = strings.TrimSuffix(op, "Handler-fm")
+	op = strings.Replace(op, "objectAPIHandlers", "s3", 1)
+	op = strings.Replace(op, "webAPIHandlers", "s3", 1)
+	op = strings.Replace(op, "adminAPIHandlers", "admin", 1)
+	op = strings.Replace(op, "(*storageRESTServer)", "internal", 1)
+	op = strings.Replace(op, "(*peerRESTServer)", "internal", 1)
+	op = strings.Replace(op, "(*lockRESTServer)", "internal", 1)
+	op = strings.Replace(op, "stsAPIHandlers", "sts", 1)
+	op = strings.Replace(op, "LivenessCheckHandler", "healthcheck", 1)
+	op = strings.Replace(op, "ReadinessCheckHandler", "healthcheck", 1)
```
Comment (harshavardhana) [inline cmd/admin-trace.go]: This type of information should come from the server we shouldn't have hardcoded entries like this in `mc` related to MinIO server. Bring these constants as part of `pkg/madmin` and move these functions there instead.

**Comment by harshavardhana** [comment]
Can you resolve conflicts @poornas ?

**Comment by poornas** [comment]
> Can you resolve conflicts @poornas ?

done.

**Comment by harshavardhana** [comment]
Can you fix the build issue @poornas 
```
Running staticcheck
cmd/admin-trace.go:73:2: var funcNameRegex is unused (U1000)
make: *** [staticcheck] Error 1
The command "make verifiers" exited with 2.
```

**Comment by poornas** [comment]
done

**Pull request merged**